src/patternfly/components/Wizard/wizard.scss

```diff
@@ -57,44 +64,54 @@
   --#{$wizard}__nav-link--before--Color: var(--pf-t--global--text--color--regular);
   --#{$wizard}__nav-link--before--FontSize: var(--pf-t--global--font--size--body--default);
-  --#{$wizard}__nav-link--before--TranslateX: calc(-100% - var(--pf-t--global--spacer--xs));
-  --#{$wizard}__nav-link--before--TranslateY: -50%;
   --#{$wizard}__nav-link--m-current--before--BackgroundColor: var(--pf-t--global--color--brand--default);
   --#{$wizard}__nav-link--m-current--before--Color: var(--pf-t--global--text--color--on-brand--default);
   --#{$wizard}__nav-link--m-disabled--before--BackgroundColor: transparent;
   --#{$wizard}__nav-link--m-disabled--before--Color: var(--pf-t--global--text--color--disabled);
 
+  // Nav link status icon
+  --#{$wizard}__nav-link-status-icon--Color: var(--pf-t--global--icon--color--regular);
+  --#{$wizard}__nav-link--m-danger__nav-link-status-icon--Color: var(--pf-t--global--icon--color--status--danger--default);
+  --#{$wizard}__nav-link-status-icon--Top: var(--pf-t--global--spacer--xs);
+  --#{$wizard}__nav-link-status-icon--FontSize: var(--pf-t--global--icon--size--font--xl);
+
   // Toggle
   --#{$wizard}__toggle--BackgroundColor: var(--pf-t--global--background--color--primary--default);
   --#{$wizard}__toggle--ZIndex: var(--pf-t--global--z-index--xs);
   --#{$wizard}__toggle--PaddingTop: var(--pf-t--global--spacer--lg);
   --#{$wizard}__toggle--PaddingRight: var(--pf-t--global--spacer--md);
   --#{$wizard}__toggle--PaddingBottom: var(--pf-t--global--spacer--lg);
-  --#{$wizard}__toggle--PaddingLeft: calc(var(--pf-t--global--spacer--lg) + var(--#{$wizard}__nav-link--before--Width) + var(--pf-t--global--spacer--sm));
+  --#{$wizard}__toggle--PaddingLeft: var(--pf-t--global--spacer--lg);
   --#{$wizard}__toggle--BorderBottomWidth: var(--pf-t--global--border--width--regular);
   --#{$wizard}__toggle--BorderBottomColor: var(--pf-t--global--border--color--default);
-  --#{$wizard}--m-in-page__toggle--xl--PaddingLeft: calc(var(--pf-t--global--spacer--xl) + var(--#{$wizard}__nav-link--before--Width) + var(--pf-t--global--spacer--sm));
 
   // Toggle number
-  --#{$wizard}__toggle-num--before--Top: #{pf-size-prem(-2px)}; // TODO this should probably be calculated
+  --#{$wizard}__toggle-num--Top: 0;
 
   // Toggle list item
   --#{$wizard}__toggle-list-item--not-last-child--MarginRight: var(--pf-t--global--spacer--sm);
   --#{$wizard}__toggle-list-item--MarginBottom: var(--pf-t--global--spacer--xs);
-
+  --#{$wizard}__toggle-list-item--Gap: var(--pf-t--global--spacer--sm);
+
+  // Toggle status icon
+  --#{$wizard}__nav-link-status-icon--LineHeight: 1;
+  --#{$wizard}__toggle-status-icon--Color: var(--pf-t--global--icon--color--regular);
+  --#{$wizard}__toggle--m-danger__nav-link-status-icon--Color: var(--pf-t--global--icon--color--status--danger--default);
+  --#{$wizard}__toggle-status-icon--Top: calc(var(--pf-t--global--spacer--xs) / 2);
```

Comment:
I could be wrong here, but it looks like this use of `spacer--xs` doesn't correspond with alignging the status-icon with something else that uses `spacer--xs` as a top spacer... this calc just returns 2px which looks to be about the adjustment that works visually to center the icon? If so, I _might_ just use `2px` here instead? It's really a nit, but if a user (or we as PF) themed/customized `spacer--xs` for a compact theme or whatever, I don't think we would want this value to change necessarily. I think the same comment applies to https://github.com/patternfly/patternfly/blob/dc3604d18826769957b3a1a6271ca095453fd333/src/patternfly/components/Wizard/wizard.scss#L82 

Kind of a philosophical question around using a token because it gives the look you're going for, versus it being the right token given the token's meaning. Since our spacer tokens are so generic, I don't think it matters, but figured I'd leave a comment for light reading  😅